jsonschema-core.md

```diff
@@ -239,6 +239,28 @@ While the empty schema object is unambiguous, there are many possible
 equivalents to the `false` schema. Using the boolean values ensures that the
 intent is clear to both human readers and implementations.
 
+#### String JSON Schemas
+
+A string schema operates as an implicit reference to another schema object.
+A string schema is a shorthand for a schema containing only the `$ref` keyword.
+
+For example, the string schema
+
+```json
+"https://example.com/schema"
+```
+
+is equivalent to
+
+```json
+{
+  "$ref": "https://example.com/schema"
+}
+```
```

Comment:
Maybe a little more descriptive example?

```json
{
  "properties": {
    "foo": "https://example.com/schema"
  }
}
```

is equivalent to

```json
{
  "properties": {
    "foo": {
      "$ref": "https://example.com/schema"
    }
  }
}
```

Reply:
Yeah, I was wondering about that.  Feedback is good.